lib/religion/religion.ts

```diff
@@ -1434,7 +1452,7 @@ export const religion: ReligionData = {
           ],
           followers: {
             description: 'As the goddess of Agriculture, Demeter has a dedictated following among anyone who farmed. She was also a major figure of worship in the Eleusinian mysteries, which promised a better afterlife to its followers.',
-            favouredWeapon: undefined,
+            favouredWeapon: 'Farmers',
```

Comment:
favoured weapon is for the weapon that the followers use- i.e. Hephaestus' followers might prefer a hammer (don't flame me I know that's wrong)

Reply:
Whoops, I got that in the wrong place